pkg/cloudprovider/vsphere/vsphere_test.go

```diff
@@ -79,13 +76,6 @@ func configFromSimWithTLS(tlsConfig *tls.Config, insecureAllowed bool) (vcfg.Con
 	model.Service.TLS = tlsConfig
 	s := model.Service.NewServer()
 
-	// STS simulator
-	path, handler := sts.New(s.URL, vpx.Setting)
```

Comment:
Need these for the LoginByToken test

Reply:
What's weird is `make test` works on my local setup... hmmmm. Will revert.